src/test/java/org/folio/circulation/resources/renewal/OverrideRenewalStrategyTest.java

```diff
@@ -255,6 +267,33 @@ public void cannotOverrideWhenOverrideDateBeforeCurrentDueDate() {
     assertThat(renewedLoan, hasValidationError(hasMessage("renewal would not change the due date")));
   }
 
+  @Test
+  public void nonLoanableAgedToLostItemShouldBeProperlyRenewed() {
```

Comment:
What does properly renewed mean?

Reply:
It means status changed to checked out and billing date and time and billing flag are removed.

Reply:
Could this test instead be renamed and expanded to be a general example of how it should be possible to renew a non loanable aged to lost item?

Reply:
Do you mean to create an API test instead?

Reply:
> Do you mean to create an API test instead?

No, I'm glad this is a unit test. Are those changes expressed in the test the only changes made by this code?

Reply:
It seems no. This PR allows correct override of mixed override scenarios for a lost item: when item is lost and policy is not loanable/renewable, etc.

Reply:
I'm not sure I follow, does that mean this PR involves changes that aren't described by this new test?

Reply:
No, this test is exactly the scenario described in jira issue.

Reply:
I think I'm a little lost. Given this fix is important, I'll resolve this conversation and approve.